server/tests/user.rs

```diff
@@ -46,6 +46,179 @@ mod user_delete_bots_mutation {
     }
 }
 
+#[cfg(test)]
+mod user_delete_regular_users_mutation {
+    use test_utils::*;
+
+    #[actix_rt::test]
+    async fn deletes_a_user_permanently() {
+        let mut transaction = begin_transaction().await;
+        let user_id: i32 = 10;
+        let deleted_user_id: i64 = 4;
+
+        let response = Message::new("UserDeleteRegularUsersMutation", json!({ "id": user_id }))
+            .execute_on(&mut transaction)
+            .await;
+
+        assert_ok(response, json!({ "success": true, })).await;
+
+        let req = Message::new("UuidQuery", json!({ "id": user_id }))
+            .execute_on(&mut transaction)
+            .await;
+
+        assert_not_found(req).await;
+
+        let deleted_got_activity = Message::new(
+            "UserActivityByTypeQuery",
+            json!({ "userId": deleted_user_id }),
+        )
+        .execute_on(&mut transaction)
+        .await;
+
+        let edits: i32 = 893;
+        let reviews: i32 = 923;
+        let comments: i32 = 154;
+        let taxonomy: i32 = 944;
+
+        assert_ok(
+            deleted_got_activity,
+            json!({ "edits": edits, "reviews": reviews, "comments": comments, "taxonomy": taxonomy,}),
+        )
+            .await;
+
+        let check_ad = sqlx::query!(r#"select author_id from ad where id = 1"#)
+            .fetch_optional(&mut transaction)
+            .await;
+
+        assert_eq!(check_ad.unwrap().unwrap().author_id, deleted_user_id);
```

Comment:
```suggestion
            .fetch_one(&mut transaction)
            .await;

        assert_eq!(check_ad.unwrap().author_id, deleted_user_id);
```
Same fot the other lines: (96, 99), (102,105) and so on.